pyinstaller.spec

```diff
@@ -0,0 +1,114 @@
+# MIT License
```

Comment:
file needs to be named pyinstaller.py otherwise the build system tries "RPM things" with it.  I think you left debug naming on this?